src/org/gk/gkCurator/authorTool/ComplexInstanceHandler.java

```diff
@@ -10,20 +10,29 @@
 
 import org.gk.model.GKInstance;
 import org.gk.model.ReactomeJavaConstants;
+import org.gk.property.SearchDBTypeHelper;
 import org.gk.render.InstanceToRenderableConverter;
 import org.gk.render.Renderable;
 import org.gk.render.RenderableComplex;
 import org.gk.render.RenderableFactory;
 
 public class ComplexInstanceHandler extends InstanceHandler {
+    private SearchDBTypeHelper typeHelper;
```

Comment:
By adding this attribute, you are begging to create a new sup-class to which ComplexInstanceHandler and EntityInstanceHandler can subclass to. Please note that code duplications.

Reply:
This is a good point! Fixed in (7f7d48d)[https://github.com/reactome/CuratorTool/pull/23/commits/7f7d48dae85ac51332a4bb83298257780b0a17de].